src/modules/components/asset/assetDataListItem/assetDataListItemStructure/assetDataListItemStructure.test.tsx

```diff
@@ -20,24 +20,10 @@ describe('<AssetDataListItem.Structure /> component', () => {
         expect(screen.getByText(props.symbol)).toBeInTheDocument();
     });
 
-    it('correctly renders amount, fiat price and price change', () => {
-        const props = { amount: 10, fiatPrice: 1250, priceChange: 25 };
+    it('correctly renders amount and fiat price', () => {
+        const props = { amount: 10, fiatPrice: 1250 };
         render(createTestComponent(props));
         expect(screen.getByText('$12.50K')).toBeInTheDocument();
         expect(screen.getByText(props.amount)).toBeInTheDocument();
-        expect(screen.getByText('+$2.50K')).toBeInTheDocument();
-        expect(screen.getByText('+25.00%')).toBeInTheDocument();
-    });
-
-    it('renders unknown with fiatPrice is not set', () => {
-        const props = { fiatPrice: undefined };
-        render(createTestComponent(props));
-        expect(screen.getByText('Unknown')).toBeInTheDocument();
-    });
```

Comment:
Is there a reason we are removing this fallback (and test) from the DLI as well?

Reply:
As far as I can tell now that we don't use price change there is no need to use the fallback as we always have an amount and USD defaults to 0 if we don't have it.

Reply:
Fair point. Let's check maybe what product thinks. 0 feels like it's inaccurate for faulty data possibility, but 'Unknown' doesn't really match any other usage. Perhaps we split the difference and use the '-' fallback, but let's check.

Reply:
Actually no, the more I think about this you are correct, I have restored this with some small additions due to overall layout shift of the DLI